src/core/scene/a-scene.js

```diff
@@ -17,6 +17,14 @@ var isMobile = utils.device.isMobile();
 var registerElement = re.registerElement;
 var warn = utils.debug('core:a-scene:warn');
 
+// Experimental versions of Chromium used for WebVR have a known issue
+// where the requestAnimationFrame timestamp is off by 1000
+// (microseconds instead of milliseconds)
+// This threshold is a time delta between requestAnimationFrame calls
+// that is implausible for correct times in milliseconds,
+// but reasonable for incorrect values in microseconds.
+var FIX_TIMESTAMPS_DELTA_THRESHOLD = 5000;
```

Comment:
This does not belong to this patch. We don't want to introduce workarounds for bugs in experimental builds